content/300-guides/100-prisma-guides/240-seed-database.mdx

```diff
@@ -0,0 +1,169 @@
+---
+title: Seeding your database with Prisma Client
+metaDescription: Learn how to seed your database using Prisma Client
+---
+
+<TopBlock>
+
+This guide describes how to seed your database using Prisma Client. This makes it really easy to ensure you can replay the necessary data in your  You may want to do this for multiple reasons:
```

Comment:
`data in your  You`
Something is missing here, `database.`?

Reply:
Pushed an actual end of sentence... 🤦